lib/cpp/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ if(WIN32)
 else()
     TARGET_LINK_LIBRARIES_THRIFT(thrift ${SYSLIBS})
 endif()
+ADD_PKGCONFIG_THRIFT(thrift)
```

Comment:
I wonder if we could just have ADD_LIBRARY_THRIFT call this with the first argument?

Reply:
The pkgconfig file-names don't match the target names. (thriftnb vs thift-nb.pc.in, etc)

I initially tried it this way, but figured being explicit was preferable to a magic find/replace.

Reply:
Also see thrift-qt5 below, where 1) the name doesn't match  and 2) the pc.in file lives in a different directory from the ADD_LIBRARY_THRIFT call.